docs/docs/releases/0.2.1.md

```diff
@@ -0,0 +1,152 @@
+# Mathesar 0.2.1
+
+## Summary
+
+Mathesar 0.2.1 addresses a number of bug fixes identified by members of our community during our **beta** release. This release also includes some quality-of-life improvements to Mathesar like nicknames for databases, persisted column widths, and the ability to connect databases without a password.
+
+We also have a new form to allow users to submit suggestions and feedback about Mathesar to our team.
+
+_This page provides a comprehensive list of all changes since version [0.2.0](./0.2.0.md)._
+
+<!-- ## Breaking changes -->
+<!-- (This section lists any breaking changes to publicly exposed and documented machine interfaces to Mathesar such as the API or DB functions) -->
+
+
+## Improvements
+<!-- (Each feature within this section should have its own level-three heading) -->
+
+### Database nicknames
+
+[#4262](https://github.com/mathesar-foundation/mathesar/pull/4262 "Frontend support for databases nicknames")
+[#4244](https://github.com/mathesar-foundation/mathesar/pull/4244 "Backend support for database nicknames")
+[#4261](https://github.com/mathesar-foundation/mathesar/pull/4261 "Allow DB name/host/port to be updated")
+
+### In-app feedback
+
+[#4241](https://github.com/mathesar-foundation/mathesar/pull/4241 "Move feedback to app header, add mailing list link to home page")
+
+### Column widths are persisted
+
+[#4257](https://github.com/mathesar-foundation/mathesar/pull/4257 "Persist column width")
+[#4248](https://github.com/mathesar-foundation/mathesar/pull/4248 "Backend support to persist customized column widths")
+
+### Docker Compose installs are updated to Postgres 17
+
+!!! info "For existing users"
+
+    Existing users who would like to update to Postgres 17 should use our new docker compose file and run `cp -r msar msar.backup` prior to upgrading Postgres in the event of issues.
+
+  New installs of Mathesar using our recommended [Docker Compose](../administration/install-via-docker-compose.md) installation will now use Postgres 17. Prior to 1.2.1 Mathesar installs configured via our docker compose file used version 13.
+
+
+[#4263](https://github.com/mathesar-foundation/mathesar/pull/4263 "Upgrade docker compose to PG 17")
+
+### Support connections without a database password
+
+[#4252](https://github.com/mathesar-foundation/mathesar/pull/4252 "Partially support passwordless Database Roles")
+[#4255](https://github.com/mathesar-foundation/mathesar/pull/4255 "Copy suggestions for passwordless roles in Mathesar")
+
+## Bug fixes
+
+- Drop schema contents when deleting [#4251](https://github.com/mathesar-foundation/mathesar/pull/4251 "Drop schema contents when deleting")
```

Comment:
I'd put this in features rather than "bug fixes".

- It's new behavior for Mathesar.
- It's novel in the field, AFAIK. No other product (that I'm aware of) has the same combination of safety and convenience provided by this.
- It warrants a bit more description, since the difference between this and `CASCADE` may surprise or interest a knowledgeable user, and may make some more confident when letting their users use Mathesar to access the DB.